src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callBack, number) {
+      return callBack(number);
```

Comment:
The 'operate' method should update the 'result' property of the 'calculator' object. Right now it just returns the result of the callback without storing it. Consider updating this method to store the result of the callback.